src/HomeScreen.js

```diff
@@ -7,45 +7,32 @@ import PlayerStats from './PlayerStats.js'
 export const socket = io(); // Connects to socket connection
 
 export default function Home ({playerName}) {
-  //state for joining multiplayer room or not
-  const [playerJoinedMultiplayer, setPlayerJoinedMultiplayer] = useState(false)
-  //state list of all players in all the rooms
-  const [activePlayers, setActivePlayers] = useState([])
-  const room = 'Multiplayer'
+  const [playerStartedGame, setPlayerStartedGame] = useState(false) // State for joining multiplayer room or not
+  const [activePlayers, setActivePlayers] = useState([]) // State list of all players in all the rooms
+  const [room, setRoom] = useState(""); // State for keeping track of the room the player is in
 
-  const leaveRoom = ()=>{
```

Comment:
Removing this because the back button from the game screen shouldn't make them leave the room on the backend. It should only show the lobby UI. From the lobby, the player should be able to leave the room they joined. However, for sprint 1, I say that we leave this out since the players don't have their own lobbies yet and everyone joins "Multiplayer".